OmniKit/PumpManager/OmnipodPumpManager.swift

```diff
@@ -31,6 +31,15 @@ public enum OmnipodPumpManagerError: Error {
     case notReadyForCannulaInsertion
 }
 
+public enum RunCommandSessionType {
```

Comment:
I understand that you're trying to reduce the number of functions (and all the boiler plate that goes with them). But the savings comes at the expense of extensibility. If any of these commands later evolve to require parameters, or worse, want to return a different type, then it starts to get messy. I recommend keeping a function based api on the pump manager.

Reply:
Done, although I have some reservations about doing it this way.